q2_dada2/assets/run_dada_single.R

```diff
@@ -188,10 +188,8 @@ seqtab <- makeSequenceTable(dds)
 
 # Remove chimeras
 cat("4) Remove chimeras (method = ", chimeraMethod, ")\n", sep="")
-if(chimeraMethod == "pooled") {
-  seqtab <- removeBimeraDenovo(seqtab, method=chimeraMethod, minFoldParentOverAbundance = minParentFold, multithread=multithread)
-} else if(chimeraMethod == "consensus") {
-  seqtab <- removeBimeraDenovo(seqtab, method=chimeraMethod, minFoldParentOverAbundance = minParentFold, multithread=multithread)
+if(chimeraMethod %in% c("pooled", "consensus")) {
```

Comment:
> `%in%`

That's a very interesting spelling.

Thanks! This is much cleaner than what I had done to your R code.